backend/backend/settings/test.py

```diff
@@ -5,5 +5,5 @@
 import logging
 logging.disable(logging.CRITICAL)
 
-LEDGER_CALL_RETRY = False
 ORG_NAME = 'OrgTestSuite'
+LEDGER_SYNC_ENABLED = True
```

Comment:
Why do you change the test setting from `LEDGER_CALL_RETRY = False` to `LEDGER_SYNC_ENABLED = True` ?

Reply:
Okay `LEDGER_SYNC_ENABLED = True` moved to test settings :)

Reply:
`LEDGER_CALL_RETRY` has no effect in the tests: it's [only used here](https://github.com/SubstraFoundation/substra-backend/commit/f0d39f4dadbfdef6a3b23c8ffc37c09ddf2e5010#diff-342b571689fc99df2ba094e899fbd715R35) and this line is never executed during test runs (tested)